appveyor.yml

```diff
@@ -0,0 +1,16 @@
+environment:
+  matrix:
+  - TARGET: x86_64-pc-windows-msvc
+  - TARGET: i686-pc-windows-msvc
+  - TARGET: i686-pc-windows-gnu
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-1.6.0-${env:TARGET}.exe"
+  - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
```

Comment:
Doh! I will have to fix that tomorrow then or you can merge and fix it after the fact
On Jan 27, 2016 23:59, ferris <notifications@github.com> wrote:In appveyor.yml:
> @@ -0,0 +1,16 @@
> +environment:
> +  matrix:
> +  - TARGET: x86_64-pc-windows-msvc
> +  - TARGET: i686-pc-windows-msvc
> +  - TARGET: i686-pc-windows-gnu
> +install:
> +  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-1.6.0-${env:TARGET}.exe"
> +  - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"

looks like this was the culprit for the failing build

—Reply to this email directly or view it on GitHub.